.github/workflows/build-wheel-linux-x86_64.yaml

```
@@ -262,7 +262,8 @@ jobs:
               -DKokkos_ENABLE_SERIAL=ON \
               -DKokkos_ENABLE_OPENMP=ON \
               -DENABLE_WARNINGS=OFF \
-              -DENABLE_OPENQASM=ON
+              -DENABLE_OPENQASM=ON \
+              -DENABLE_OPENMP=ON
```

We'll add OMP when packaging our wheels with this lib.
```suggestion
              -DENABLE_OPENQASM=ON \
              -DENABLE_OPENMP=OFF
```

---

My goal was only to make implicit flags explicit. I checked the lightning repo, and I believe `ON` is the default there (assuming CMake can find an OpenMP package):
https://github.com/PennyLaneAI/pennylane-lightning/blob/74427014b0e7ddc72c33bbadeda53bd97b70bdd5/CMakeLists.txt#L51-L53

Since we are already turning OpenMP on in the Lightning Kokkos config (with the flag `-DKokkos_ENABLE_OPENMP=ON`, I figured we must have already been building the linux wheels with OpenMP enabled on the qubit device.

What do you think?

---

I like the idea of being explicit with these CMake options. The last time Erick and I checked the default value of OMP in lightning.qubit on Linux was OFF. They maybe updated it in the new release! Given the fact that we're not shipping our wheels with the OMP library on Linux and we haven't had no complain so far, it maybe safe to keep it ON.